programs/programs/co/collegeinvest_first_step/calculator.py

```
@@ -0,0 +1,28 @@
+
+    No income limit. Data gaps: birth/adoption location (CO residency used as proxy)
+    and the military exception (active military families with permanent CO residence
+    qualify even if the child was born out of state) are not verifiable from screener
+    data; both gaps are noted in the program description.
+    """
+
+    member_amount = 121
+    max_age = 7
+    min_birth_year = 2020
+    child_relationships = ["child", "stepChild", "fosterChild", "grandChild"]
+    dependencies = ["age", "relationship"]
```

Ruff (0.15.13)

[warning] Mutable default value for class attribute

(RUF012)